static/bidder-info/richaudience.yaml

```diff
@@ -0,0 +1,15 @@
+maintainer:
+  email: partnerintegrations@richaudience.com
+gvlVendorID: 108
+gdpr:
+  timeouts_ms:
+    init_vendorlist_fetches: 5000
+    active_vendorlist_fetch: 5000
```

Comment:
Please remove the `gdpr` section. This is not valid for bidder yaml files.

Reply:
in next commit i have solved